guacamole-docker/README.md

```diff
@@ -115,6 +119,35 @@ The process for doing this via the `mysql` utility included with MySQL is
 documented in
 [the Guacamole manual](http://guacamole.incubator.apache.org/doc/gug/jdbc-auth.html#jdbc-auth-mysql).
 
+
+Deploying Aoache Guacamole with LDAP authentication
+--------------------------------------------------
+
+    docker run --name some-guacamole --link some-guacd:guacd    \
+        -e LDAP_HOSTNAME=172.17.42.1                            \
+        -e LDAP_USER_BASE_DN=ou=people,dc=example,dc=com        \
+        -e LDAP_CONFIG_BASE_DN=ou=connections,dc=example,dc=com \
+        -d -p 8080:8080 glyptodon/guacamole
+
+Linking Guacamole to your LDAP directory will require additional configuration parameters
```

Comment:
Looks like I made this mistake, too, within the manual:

~~~xml
<para>Linking Guacamole to your LDAP directory will require additional configuration
  parameters specified via environment variables. These variables collectively
  describe how Guacamole will connect to LDAP:</para>
~~~

Unless you think the wording is OK as-is.